packages/core/src/Sheets/Domain/Range.ts

```diff
@@ -1,8 +1,9 @@
-import { ICurrentUniverService } from 'src/Service/Current.service';
+import { Inject } from '@wendellhu/redi';
+import { IDCurrentUniverService, ICurrentUniverService } from '../../Service/Current.service';
 
 // TODO@wzhudev: here we still have some
 
-import { Inject } from '@wendellhu/redi';
+// eslint-disable-next-line import/no-cycle
```

Comment:
Please do not silent this warning. We intent to fix this later.

Reply:
fixed